src/scheduler/Scheduler.java

```diff
@@ -72,7 +72,7 @@ private void executeProcess(PCB processPCB) {
                     processPCB.setWaitTo2();
                     ProcessList.addBlockedProcess(processPCB);
                 } else {
```

Comment:
The round robin algorithm does not need lines 66 and 67, and line 79 should be replaced by `ProcessList.addReadyProcessInLastPosition (processPCB);` for the round robin algorithm. Anything look like I did in the createRoundRobin branch